keepassxc-browser/content/keepassxc-browser.js

```diff
@@ -513,6 +513,19 @@ kpxc.prepareCredentials = async function() {
 
     kpxc.initLoginPopup();
     kpxc.initAutocomplete();
+
+    if (kpxc.settings.autoFillRelevantCredential) {
+        const pageUuid = await sendMessage('page_get_login_id');
+        if (pageUuid) {
+            kpxc.credentials.some(c => {
+                if (c.uuid === pageUuid) {
```

Comment:
You could just replace this with a `kpxc.credentials.find()`, and if a credential is found you get the correct combination for the fill.
Is there some reason why the last combination (or -1 as index if there's no combinations) is returned? There could be multiple combinations available.

Reply:
changed `some` to `kpxc.credentials.find`